packages/ra-core/src/form/choices/useChoices.tsx

```diff
@@ -27,6 +27,8 @@ export interface UseChoicesOptions {
     optionText?: OptionText;
     disableValue?: string;
     translateChoice?: boolean;
+    createValue?: string;
+    createHintValue?: string;
```

Comment:
Aren't you introducing _new_ props indirectly to the `<AutocompleteInput>` and `<SelectInput>`? That should target the `next` branch

Reply:
Technically you're correct, but:
- For `<AutocompleteInput>` the props were already there, but non-documented
- For `<SelectInput>` the `createValue` prop was already there (non-documented), only the `createHintValue` prop is new
In both cases, these are undocumented props, so no need for a minor version IMO.